tests/test_dummy_animal_module.py

```diff
@@ -0,0 +1,61 @@
+"""Test module for dummy_animal_module.py.
+
+This module tests the functionality of dummy_animal_module.py
+"""
+
+# Problems:
+# handling string returns for details
+# handling initialization of animals with zero energy
+# test that eat can't drive energy negative
+
+import virtual_rainforest.animals.dummy_animal_module as am
+
+
+class TestPlant:
+    """Test Plant class."""
+
+    def test_plant_growth0(self):
+        """Testing plant_growth at 100% energy."""
+        p = am.Plant("tree", 10.0, 1)
+        print("testing plant growth : plant")
```

Comment:
I assumed you had (hope I didn't sound snarky!). I just wanted to double-check that you weren't testing in some roundabout way instead, e.g. by importing these functions into a script and running it, which would explain the `print` statements.

Reply:
The print statements got in there because I only kinda knew how to set up this sort of testing and was following online tutorials that used them! You didn't sound snarky at all.

Reply:
With respect to putting dummy classes in other people's modules, now that we're shifting to module connections I'll bring it up as a early step. Thanks for the suggestion.